src/components/PreferencesNav.vue

```diff
@@ -0,0 +1,48 @@
+<script lang="ts">
+import Vue from 'vue';
+import PreferencesNavItem from '@/components/PreferencesNavItem.vue';
+
+export default Vue.extend({
+  name:       'preferences-nav',
+  components: { NavItem: PreferencesNavItem },
+  props:      {
+    currentNavItem: {
+      type:    String,
+      default: 'Application'
```

Comment:
Do we need a hard-coded default here? (We'll totally forget to change it if we ever change the first item.)

Reply:
Remove default and made prop required.